lib/index.js

```diff
@@ -92,12 +94,48 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
         });
     }));
 
+  app.get(
+    adminPath,
+    ensureAuthenticated,
+    intParser(),
+    validate({query: 'account.pagination'}),
+    asyncHandler(async (req, res) => {
+      const {email, cursor = null, limit = 10} = req.query;
+      const {actor} = req.user || {};
+      const query = {'account.email': email};
+      if(cursor) {
+        query['account.id'] = {$gt: cursor};
+      }
+      const options = {limit, sort: {'account.id': -1}};
+      const records = await brAccount.getAll({actor, query, options});
+      res.status(200).json(records);
```

Comment:
if I don't say it, somone else will, 200 is implicit with just `res.json()`

Reply:
done